notifications.json

```diff
@@ -0,0 +1,861 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "1.0",
+        "title": "Notification API",
+        "description": "Notification API"
+    },
+    "host": "e2effzjxvtgbmuilw-mock.stoplight-proxy.io",
+    "basePath": "/v1",
+    "schemes": [
+        "https"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/Messages/{id}": {
```

Comment:
also here we're talking specifically about notifications, we should avoid ambiguity and try to be consistent, messages is indeed a broader concept that implies a conversation - in our case the communication is unidirectional, so it's notifications